scenarios/kubernetes_kops_aws.py

```diff
@@ -168,6 +187,17 @@ def main(args):
         '--kops-ssh-key=%s' % aws_ssh,
     ]
 
+    if args.build is not None:
```

Comment:
ah, you are right. Copy the same thing over, since later on we will shuffle everything here back.